### ISSUE TITLE
Change generated URI for value sets and code systems

### ISSUE THREAD

**Issue by cmoesel** [standardhealth/shr-text-import#6]
Currently, the generated URIs for value sets and code systems match where the vs/cs would be found in the FHIR IG (e.g., http://standardhealthrecord.org/fhir/ValueSet/shr-allergy-AllergenIrritantVS).  We've determined that we'd prefer the canonical URI not to be tied to FHIR, so we should switch them over to a more shr-ish URI (e.g., http://standardhealthrecord.org/shr/allergy/vs/AllergenIrritantVS).

**Comment by cmoesel** [comment]
Unfortunately, the FHIR IG publisher _requires_ value set URIs to conform to a particular pattern (`${canonical-ig-url}/ValueSet/${value-set-id}`).  Attempting to define our own URI outside of that pattern results errors (like the following) from the IG publisher:
```
Publishing Content Failed: URL Mismatch http://standardhealthrecord.org/fhir/ValueSet/shr-actor-AgeGroupVS vs http://standardhealthrecord.org/shr/actor/vs/AgeGroupVS
```

This is apparently _by design_.  The notion being that if you are defining a value set using FHIR, then its URI should be resolvable within the IG that defines it.  See:
https://chat.fhir.org/#narrow/stream/implementers/topic/ValueSet.20Url.20Validation.20During.20IG.20generation

As I see it, we have four potential approaches to this problem:
1. Remain as-is, with SHR value sets and code systems having canonical URIs following the above pattern.
2. Change the canonical URI of our IG to be _http://standardhealthrecord.org/_ instead of _http://standardhealthrecord.org/fhir/_.  Then at least our value sets and code systems wouldn't have the word "fhir" in them.  But... it would also mean that we _should_ host the IG documentation at that same path (the root of our server) -- which is not ideal.
3. Don't include FHIR definitions of the Value Sets and Code Systems in the IG.  We would treat the Value Sets and Code Systems in the IG as externally defined value sets and code systems -- and we would then need to choose how we want to formally define and host them elsewhere (outside the IG).  This means, however, that the IG is not a fully self-contained package of SHR definitions.
4. Don't use the FHIR IG Publisher.  Create our own.  This would be a significant effort.

**Comment by cmoesel** [comment]
As it turns out, it is extremely simple to modify the IG Publisher code to stop this behavior (it's just one line of code, and then rebuild it).  So... we also have that option.

**Comment by cmoesel** [comment]
Checked in the code to define our own value set URIs, and it is in the grammar_4.2 branch.  Currently using a patched version of the FHIR 1.8.0 igpublisher.jar, but a new build of the igpublisher.jar (ever so slightly post-3.0) adds support for arbitrary URIs too.  Once we switch to that, we won't need a patched version.

**Comment by cmoesel** [comment]
Merged.
